api/scripts/wazuh_apid.py

```diff
@@ -72,10 +72,10 @@ def configure_ssl(params: dict, config: APISSLConfig):
                         'Attempting to generate them')
             private_key = generate_private_key(config.key)
             logger.info(
-                f"Generated private key file in WAZUH_PATH/{to_relative_path(config.key)}")
+                f"Generated private key file in {config.key}")
```

Comment:
If the user sets a relative path in the `config.key` the message will be misleading. I would prefer to get the absolute path and show that instead.

If this doesn't involve a single line change then I would leave it as a technical debt though.

Reply:
Because it is a passed string and not a structure like the ones we are using from pathlib, I think the change would be more than 1 line of code. I add to the technical debt to review the paths in the error messages as well as make this format consistent.